compiler/il/OMRNode.cpp

```diff
@@ -7529,14 +7529,33 @@ OMR::Node::setIsOSRGuard()
       _flags.set(osrGuard);
    }
 
+void
+OMR::Node::setIsBreakpointGuard()
+   {
+   TR::Compilation *c = TR::comp();
+   TR_ASSERT(self()->getOpCode().isIf(), "assertion failure");
+   if (performNodeTransformation1(c, "O^O NODE FLAGS: Setting fsdGuard flag on node %p\n", self()))
+      _flags.set(fsdGuard);
```

Comment:
Minor question, is there a reason why the flag bit is called "fsdGuard" whereas the routine calls it a "breakpoint guard" ? i.e. why not make it consistent ?